es6-promises/main.js

```diff
@@ -0,0 +1,11 @@
+import takeAChance from './take-a-chance.js';
+
+const promi = takeAChance('Lamberto');
+
+promi.then(resolve => {
+  console.log(resolve);
+});
+
+promi.catch(reject => {
+  console.log(reject.message);
+});
```

Comment:
To properly handle errors, these need to be chained:
```
promi.then(...).catch(...);
```
Usually you would skip the intermediate variable and just use:
```
takeAChance('Lamberto')
  .then(...)
  .catch(...);
```

Reply:
ahhh ok i understand